recipes/freefem-feedstock/meta.yaml

```diff
@@ -0,0 +1,104 @@
+{% set name = "FreeFem" %}
+{% set version = "4.11" %}
+{% set sha256 = "d0c6921791e5f94646d8dde4d9ed3c11b979e47e7bbb3c0a66467b04dd56983a" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}-sources/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build }}
+  skip: true  # [py2k or osx or win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+  host:
+    - cmake
+    - autoconf
+    - automake
+    - bison
+    - cmake
+    - flex
+    - m4
+    - make
+    - patch
+    - perl
+    - unzip    
+    - libgomp      # [linux]
+    - llvm-openmp  # [osx]   
+    - hdf5
+    - liblapack
+    - freeglut
+    - nlopt    
+    - suitesparse 
+    - libopenblas  
+    - metis
+    - petsc
+    - gsl
+    - gmm
+    - blis
+    - scalapack
+    - scikit-umfpack
+    - superlu
+    - ipopt
+    - parmetis
+    - arpack      
+    - tetgen   
+  run:
+    - libgomp      # [linux]
+    - llvm-openmp  # [osx]   
+    - hdf5
+    - liblapack
+    - libopenblas
+    - metis
+    - petsc
+    - scalapack
+    - scikit-umfpack
+    - freeglut    
+    - superlu
+    - gsl
+    - gmm
+    - blis
+    - ipopt
+    - nlopt
+    - parmetis
+    - suitesparse
+    - arpack      
+    - tetgen       
+
+test:
+  commands:
+    - test -f $PREFIX/bin/FreeFem++  # [unix]
+    - test -f $PREFIX/bin/FreeFem++-nw  # [unix]
+    - test -f $PREFIX/bin/ff-c++  # [unix]
+    - test -f $PREFIX/lib/ff++/{{ version }}/include/ff++.hpp  # [unix]
```

Comment:
You are right, I had to remove libraries because conda lib gives out errors when repackaging (it finds some link to a system library, maybe I should set a sysroot ?) so I removed the headers test too. Maybe in the future we could restore the library distribution but I will need a bit of help on this...

Reply:
I'm not exactly sure what's going on with this build system. Just guessing, but it may have some hardcoded paths in the build files and the installation of libraries to a non-standard path (/lib/ff++/${PKG_VERSION}/lib/)?